docs/language/learn-ql/ql-training.rst

```diff
@@ -1,19 +1,19 @@
-QL training and variant analysis examples
-#########################################
+TEST training and variant analysis examples
+#############################################
 
-QL and variant analysis
-=======================
+Variant analysis using TEST
+=============================
 
 `Variant analysis <https://semmle.com/variant-analysis>`__ is the process of using a known vulnerability as a seed to find similar problems in your code. Security engineers typically perform variant analysis to identify possible vulnerabilities and to ensure that these threats are properly fixed across multiple code bases.
 
-`QL <https://semmle.com/ql>`__ is Semmle's variant analysis engine, and it is also the technology that underpins LGTM, Semmle's community driven security analysis platform. Together, QL and LGTM provide continuous monitoring and scalable variant analysis for your projects, even if you don’t have your own team of dedicated security engineers. You can read more about using QL and LGTM in variant analysis in the `Semmle blog <https://blog.semmle.com/tags/variant-analysis>`__.
+`TEST <https://semmle.com/ql>`__ is the query language used by Semmle's `variant analysis <https://semmle.com/variant-analysis>`__ engine, and it is also the technology that underpins LGTM, Semmle's community driven security analysis platform. Together, TEST and LGTM provide continuous monitoring and scalable variant analysis for your projects, even if you don’t have your own team of dedicated security engineers. You can read more about using TEST and LGTM in variant analysis in the `Semmle blog <https://blog.semmle.com/tags/variant-analysis>`__.
```

Comment:
Probably needs to be rewritten to avoid making it confusing. I think that this change is probably wrong.

Reply:
I agree.